tendrl/node_agent/manager/pull_hardware_inventory.py

```diff
@@ -122,6 +124,46 @@ def getTendrlContext():
     return tendrl_context
 
 
+def get_node_network():
+    '''returns structure
+
+    {"interface_name": {"ipv4": ["ipv4address", ...],
+                        "netmask": ["subnet", ...],
+                        "status":"up/down"}, ...}
+    '''
+    interfaces = ni.interfaces()
+    rv = {}
+    invalid = ['lo', 'virbr0', 'virbr0-nic']
+    for interface in interfaces:
+        if interface not in invalid:
+            IPv4 = []
+            netmask = []
+            if is_interface_up(interface):
+                ipv4_addr_list = ni.ifaddresses(interface)[AF_INET]
```

Comment:
Dont import individual items from a module, it is confusing to see just "AF_INET", use complete names "netifaces.AF_INET" or "ni.AF_NET"